readme.md

```diff
@@ -0,0 +1,24 @@
+
+# Application Passwords
+
+![](https://cldup.com/50AyeqtUEk.png)
+
+This is a feature plugin that is a spinoff of the main Two-Factor Authentication plugin, found at https://github.com/georgestephanis/two-factor/.  With Application Passwords you are able to authenicate a user without providing his or her password directing, instead you will use a base64 encoded string of their username and a new application password.
+
+# Create a New Password and Use it
+
+1. Go to your User Profile page.
+2. Scroll down until you see the Application Passwords section.
+3. Type in a new password name (only used for describing a password, not for authenticating) and hit Add New.
```

Comment:
I haven't seen this UI, but is this essentially an "application name"? (though I assume you can use the same password with multiple apps, so this is probably more accurate)